build.gradle

```diff
@@ -36,6 +36,7 @@ ext {
     TARGET_SDK_VERSION = 28
     COMPILE_SDK_VERSION = 28
 
+    androidXLibrarayVersion = '1.0.0'
```

Comment:
Typo ;)

Reply:
Damn :D That happens when I just copy :D